app.ts

```diff
@@ -0,0 +1,87 @@
+﻿/// <reference path="scripts/typings/jquery/jquery.d.ts" />
+
+class RecordsApp {
+	numberOfItems: number;
+	pageNumber: number;
+	pageSize: number;
+	baseUrl = "http://localhost:2050/";
+
+	constructor(pageSize: number, pageNumber: number) {
+		this.pageNumber = pageNumber;
+		this.pageSize = pageSize;
+	}
+
+	// Load all columns returned from the server
+	private loadColumns(data) {
+		var header = document.createElement("thead");
+		const headerRow = document.createElement("tr");
+
+		// Add data items to the row
+		$.each(data,
+			(key, val) => {
+				var headerItem = document.createElement("th");
+				headerItem.innerHTML = String(val);
+				header.appendChild(headerItem);
+			});
+
+		header.appendChild(headerRow);
+		$("#recordsTable").append(header);
+	}
+
+	// Process and render the data that is coming from the server
+	private loadRecords(data) {
+		// Clear the table body before adding new content
+		$("#recordsTable").children("tbody").html("");
+		var tableBody = document.createElement("tbody");
+
+		// Add data items to the row
+		$.each(data,
+			(key, val) => {
+				var tableRow = document.createElement("tr");
+				tableRow.innerHTML =
+					`<td>${val[0]}</td><td>${val[1]}</td><td>${val[2]}</td><td>${val[3]}</td><td>${val[4]
+					}</td><td>${val[5]}</td><td>${val[6]}</td><td>${val[7]}</td><td>${val[8]
+					}</td><td>${val[9]}</td><td>${val[10]}</td>`;
```

Comment:
I don't think this would work if you received more or less columns from the server.